api/app/Http/Traits/RespondsWithJwt.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Traits;
+
+use Illuminate\Support\Facades\Auth;
+
+trait RespondsWithJwt
+{
+  /**
+   * Get the token array structure.
+   *
+   * @param string $token
+   *
+   * @return \Illuminate\Http\JsonResponse
+   */
+  protected function respondWithToken($token)
+  {
+    return response()->json([
+      'access_token' => $token,
+      'token_type' => 'bearer',
+      'expires_in' => Auth::factory()->getTTL() * 60
```

Comment:
`Auth::factory()->getTTL() * 60` how many minutes is that?

Reply:
By default 60, you can change it in config file, ttl is set in seconds, in this manner - 3600